core-backend/common/src/lib.rs

```diff
@@ -26,7 +26,10 @@ pub mod lazy_pages;
 
 mod utils;
 
-#[cfg(feature = "mock")]
+#[cfg(test)]
+pub(crate) mod mock;
+
+#[cfg(all(not(test), feature = "mock"))]
```

Comment:
What's the reason to make it that way?
```suggestion
#[cfg(any(feature = "mock", test))]
```
Cfg(test) cannot be thrown if lib used as external dependency

Reply:
That looks better. Thank you. I have fixed it